TrendyTravel/Views/Destinations/Details/PopularDestinationsDetailsView.swift

```diff
@@ -60,6 +60,7 @@ struct PopularDestinationsDetailsView: View {
     
     var body: some View {
         ScrollView(showsIndicators: false) {
+            // TODO: Fix proper city images
```

Comment:
peut être crée un ticket ?

Reply:
Il y en a déjà un dans notre Trello : "In Destination Details View, Photos need to be linked to the current city" 😊